public/404.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
Whats this file about?

Reply:
The `404.html` template gets served whenever there are no routes for a particular URL.

Reply:
How is this configured?

Reply:
Good on able to say you not certain. You'll learn this later :)